src/response.rs

```diff
@@ -54,7 +80,39 @@ impl Response {
         }
     }
 
-    /// Get the `StatusCode`.
+    /// Get the `StatusCode` of this `Response`.
+    ///
+    /// ```rust
+    /// # extern crate reqwest;
+    /// # #[macro_use] extern crate error_chain;
+    /// # use reqwest::{Client, StatusCode};
+    /// # use reqwest::header::Location;
+    /// #
+    /// # error_chain! {
+    /// #     foreign_links {
+    /// #         Reqwest(reqwest::Error);
+    /// #     }
+    /// # }
+    /// #
+    /// # fn run() -> Result<()> {
+    /// let client = Client::new()?;
+    /// let resp = client.post("http://httpbin.org/post")?.send()?;
+    /// match *resp.status() {
+    ///     StatusCode::Ok => println!("All good!"),
+    ///     StatusCode::Found => {
+    ///         let url = resp.headers().get::<Location>();
+    ///         println!("Received redirect to: {:?}", url);
+    ///     }
+    ///     status => bail!("Http Error, Response status: {:?}", status),
```

Comment:
Yeah that's probably a good idea. I'll see what I can do about removing error-chain requirements/usage.